Content.Server/Procedural/DungeonJob.PostGen.cs

```diff
@@ -191,7 +192,7 @@ private async Task PostGen(BoundaryWallPostGen gen, Dungeon dungeon, EntityUid g
             if (!_anchorable.TileFree(grid, neighbor, DungeonSystem.CollisionLayer, DungeonSystem.CollisionMask))
                 continue;
 
-            tiles.Add((neighbor, _tileDefManager.GetVariantTile(tileDef, random)));
+            tiles.Add((neighbor, _tile.GetVariantTile((ContentTileDefinition) tileDef, random)));
         }
```

Comment:
Isn't adding all these casts going to be really slow.

Reply:
i'm not really sure but unless you want to move the data to the base ITileDefinition there isn't really a good way to do it.